mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -399,110 +422,241 @@ def StartVisualizePointCloudTask(self, vminitparams, locationName=None, sensorSe
-        log.info('command is stopped')
-        return response
-    
+        """Cancels the current command.
+
+        Args:
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+        """
+        command = {
+            'command': 'Cancel',
+        }
+        return self._SendConfiguration(command, timeout=timeout)
+
     def Quit(self, timeout=2.0):
         # type: (float) -> Optional[Dict]
-        log.info('stopping visionserver...')
-        response = self._SendConfiguration({"command": "Quit"}, timeout=timeout)
-        log.info('visionserver is stopped')
-        return response
-    
+        """Quits the visionmanager.
+
+        Args:
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+        """
+        command = {
+            'command': 'Quit',
+        }
+        return self._SendConfiguration(command, timeout=timeout)
+
     def GetTaskStateService(self, taskId=None, cycleIndex=None, taskType=None, timeout=4.0):
-        command = {"command": "GetTaskState"}
-        if taskId:
+        """Gets the task state of the visionmanager.
+
+        Args:
+            taskId (str, optional): If specified, the taskId to retrieve the task state of.
+            cycleIndex (str, optional): Unique cycle index string for tracking, backing up, and differentiating cycles.
+            taskType (str, optional): The taskType for which the status was requested
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 4.0)
+
+        Returns:
+            dict: A dictionary with the structure:
+
+            taskParameters (dict): describes the task specific parameters if present, eg. detection params, execution verification params..
+            initializeTaskMS (int): timestamp at which the task was received and initialized , in ms (linux epoch)
+            isStopTask (bool): True if task is currently running
+            scenepk (str): scene file name
+            taskId (str): The taskId for which the status was requested
+            taskStatus (str): status of the task
+            taskStatusMessage (str): describes the task status
+            taskType (str): The task type for which the status was requested
+        """
+        command = {
+            'command': 'GetTaskState',
+        }
+        if taskId is not None:
             command['taskId'] = taskId
-        if cycleIndex:
+        if cycleIndex is not None:
             command['cycleIndex'] = cycleIndex
-        if taskType:
+        if taskType is not None:
             command['taskType'] = taskType
-        response = self._SendConfiguration(command, timeout=timeout)
-        return response
-    
+        return self._SendConfiguration(command, timeout=timeout)
+
     def GetPublishedStateService(self, timeout=4.0):
```

Comment:
No docstring?

Reply:
Added, but I don't have a proper description for it yet, so it's just the skeleton.